docs/source/en/modular_transformers.md

```diff
@@ -59,7 +64,7 @@ file, and the corresponding files will be created for you.
 
 ### Enforcement
 
-[TODO] We are introducing a new test, that makes sure the generated content matches what is present in the `modular_xxxx.py`
+You can run `python utils/check_modular_conversion.py --files src/transformers/models/<your_model>/modular_<your_model>.py` to make sure the generated content matches what is present in `modular_<your_model>.py`
```

Comment:
I'm not sure if this is accurate since it fails in my case (#35320).
I'm assuming this is the check the TODO is referring to, please double check.